server/src/test/java/org/cloudfoundry/identity/uaa/impl/config/AwsSecretsLoaderTest.java

```diff
@@ -138,7 +148,54 @@ void testCreateResourcesWithWhitespaceInSecretName() {
     }
 
     @Test
-    void testEnvironmentVariablesLoadedFromSecretName() throws JsonProcessingException {
+    void testEnvironmentVariablesLoadedFromSecretNameWithYamlContent() throws JsonProcessingException {
```

Comment:
testEnvironmentVariablesLoadedFromSecretNameWithYamlContent() and testEnvironmentVariablesLoadedFromSecretNameWithJsonContent() could be converted into parameterized test

```
@ParameterizedTest(name = "testEnvironmentVariablesLoadedFromSecretNameWithValidContent(): {index}")
@MethodSource("getValidContent")
void testEnvironmentVariablesLoadedFromSecretNameWithValidContent(List<Resource> resources) throws JsonProcessingException {
...
}

private static Stream<List<Resource>> getValidContent() throws JsonProcessingException {
      return Stream.of(resourcesWithValidYamlContent(), resourcesWithValidJsonContent());
}
```

Reply:
Fixed by Irina